src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/config/ContextInitializer.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+import com.github.tomakehurst.wiremock.common.Slf4jNotifier;
+import com.github.tomakehurst.wiremock.core.Options;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.Profiles;
+
+import static org.springframework.util.SocketUtils.findAvailableTcpPort;
+
+@Configuration
+public class ContextInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
```

Comment:
can we not use `@AutoConfigureWireMock` in order to be able to use Wiremock in your test ?